### ISSUE TITLE
make a snap stable release

### ISSUE THREAD

**Issue by elopio** [tldr-pages/tldr-node-client#176]
I've been using (and enjoying) the tldr snap for some weeks already. It seems pretty stable to me, so why don't you make a release to the stable channel?

Bonus points if you add an icon and a screenshot in https://dashboard.snapcraft.io, because then it will also appear on searches on the gnome software app. Install tldr with one click! :)

## Expected behaviour
`sudo snap install tldr` should give me the most recent stable release of tldr.
`snap find "man pages"` should show tldr in the results
searching in gnome software should show tldr in the results. 

## Actual behaviour
tldr is only installable from the edge channel.

## Environment
- Operating system - Ubuntu 18.04

**Comment by agnivade** [comment]
Done

**Comment by elopio** [comment]
Awesome!

![ekrankopio de 2017-11-23 12-44-15](https://user-images.githubusercontent.com/617831/33185984-0aecd794-d04c-11e7-8b7a-833652f7951a.png)
